QtCollider/style/ProxyStyle.cpp

```diff
@@ -33,7 +33,7 @@ void ProxyStyle::drawComplexControl ( ComplexControl ctrl, const QStyleOptionCom
 {
   // FIXME: this is a workaround for the WebKit bug #104116 (or a variation on it).
   if( ctrl == QStyle::CC_ScrollBar
-      && qobject_cast<const QWebView*>(w) != 0
+      && qobject_cast<const QWebEngineView*>(w) != 0
```

Comment:
Still wonder if this is needed anymore?

Reply:
Let's get this PR in, and try removing it in a separate change so we can A-B the behavior more easily. I recall this bug being fairly subtle, and we'll need to test on all platforms.